src/test/SafeCastLib.t.sol

```diff
@@ -200,6 +205,12 @@ contract SafeCastLibTest is DSTestPlus {
         SafeCastLib.safeCastTo24(x);
     }
 
+    function testFailSafeCastTo16(uint256 x) public {
```

Comment:
missing a `testSafeCastTo16`

Reply:
https://github.com/transmissions11/solmate/pull/311/files#diff-1cf1e7c9ff842ab06001df49e0dae514a7141c6626a24ca4f5cafd887b4d4d09R54 ? or am i an idiot and completely misunderstanding

Reply:
you've got a concrete success test, but need a successful fuzz test too hehe

Reply:
i hate when i'm an idiot - added the test